src/hotspot/cpu/ppc/frame_ppc.hpp

```diff
@@ -379,6 +386,9 @@
   int offset_fp() const         { assert_offset();  return _offset_fp; }
   void set_offset_fp(int value) { assert_on_heap(); _offset_fp = value; }
 
+  // Mark a frame as not fully initialized
```

Comment:
Maybe add a comment like "Must not be used for frames in the valid back chain."?

Reply:
Thanks for the feedback. I've added the comment and updated the Copyright years.